lib/streamlit/elements/image.py

```diff
@@ -287,7 +291,12 @@ def image_to_url(
     output_format: OutputFormat,
     image_id: str,
     allow_emoji: bool = False,
-):
+) -> str:
+    """Return a URL that an image can be served from.
+    If `image` is already a URL, return it unmodified.
+    If `image` is an emoji, return it unmodified.
```

Comment:
nit: we have [some capabilities](https://github.com/streamlit/streamlit/blob/cb35c657491d1fece6210474b95bbd4d576b3eae/lib/streamlit/string_util.py#L44) now to check if a string is actually an `emoji`. Not sure if we can apply it here directly, but it might be useful to use it here.

Reply:
we can also just add it to the tech debt DB to look into this later.

Reply:
Ah, good call! I'll make a separate PR for that